lidar_litev3hp_sensor2/lidar_litev3hp_sensor2.ino

```diff
@@ -0,0 +1,77 @@
+/*------------------------------------------------------------------------------
+
+  Driver code is based on Garmin LIDAR-Lite v3hp Arduino Library (https://github.com/garmin/LIDARLite_Arduino_Library) 
+  Setup Documentation Hardware: https://support.garmin.com/en-US/?partNumber=010-01722-00&tab=manuals
+  Setup Documentation Software: https://github.com/garmin/LIDARLite_Arduino_Library
+  Default Address:  0x62.
+
+ 
+ * Copyright (C) 2019 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ 
+------------------------------------------------------------------------------*/
+#include <LIDARLite.h>
+#include <Wire.h>
+#include <ros.h>
+#include <ros/time.h>
+#include <sensor_msgs/Range.h>
+//LIDARLite object
+LIDARLite lidarv3hp;
+//ROS node handle
+ros::NodeHandle  n;
+//Message type sensor_msgs::Range
+sensor_msgs::Range distance_msg;
+//ROS Publisher object
+ros::Publisher pub_distance( "sensor2_data", &distance_msg);
```

Comment:
This topic name should be more descriptive. Something like "range_sensor2/range"